src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java

```diff
@@ -91,6 +110,11 @@
     private String basenameSeparator;
 
     /**
+     * Language codes of translation files (de, pt, ja, etc).
+     */
+    private SortedSet<String> checkedLanguageCodes = Sets.newTreeSet();
```

Comment:
@mkordas 
Yes. why not? Sets.newTreeSet() returns a new, empty set. By default we cannot imagine what kind translations are needed.
Collections.EMPTY_SORTED_SET is not supported in Java 1.7 (marked as since 1.8).